packages/edit-site/src/components/create-template-part-modal/index.js

```diff
@@ -39,17 +39,21 @@ import {
 } from '../../utils/template-part-create';
 
 export default function CreateTemplatePartModal( {
-	closeModal,
+	area: areaProp = TEMPLATE_PART_AREA_DEFAULT_CATEGORY,
 	blocks = [],
+	buttonLabel = __( 'Create' ),
+	closeModal,
+	modalTitle = __( 'Create template part' ),
 	onCreate,
 	onError,
+	title: titleProp = '',
```

Comment:
Nit: The `title` and `area` prop could be renamed to `defaultTitle` and `defaultArea` to better communicate that they are initial values and not controlled values.